### DISCUSSION TITLE
Welcome to hotkeysplus-obsidian Discussions!

### DISCUSSION THREAD

**Discussion by argenos** [argenos/hotkeysplus-obsidian#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DouglasDahlia779** [comment]
Great plugin! Thanks so much. Your CNTRL + N to remove extra line feeds works great to clean up OCR text. You might like add it to the help screen in Obsidian, as it is not discussed there. I spent a week trying to figure out why Obsidian removed the feature in v 1.0 :-) I discovered it in your great plugin that I shut down because of V.1.0 testing. Thanks again!
